af3_mmseqs2/processoutput/alphafold3.py

```diff
@@ -0,0 +1,7 @@
+# from af3_mmseqs2.processoutput.utils import (
```

Comment:
What's this code?

Reply:
WIP, It's what I'm working on today but though it was better to get this branch merged for it's other utilities

Reply:
Code I'm working on today, can be removed for now but it will be used. WIP I guess